src/libs/conduit/conduit_utils.hpp

```diff
@@ -273,10 +273,10 @@ namespace utils
     //-----------------------------------------------------------------------------
     /// Function types for memory handlers
     //-----------------------------------------------------------------------------
-    typedef std::function<void(void*, const void*, size_t)> Handle_Memcpy;
-    typedef std::function<void(void*, int, size_t)> Handle_Memset;
-    typedef std::function<void*(size_t, size_t)> Handle_Alloc;
-    typedef std::function<void(void*)> Handle_Free;
+    typedef std::function5<void(void*, const void*, size_t)> handle_memcpy_type;
```

Comment:
```suggestion
    typedef std::function<void(void*, const void*, size_t)> handle_memcpy_type;
```
looks like a 5 sneaked in

Reply:
woops---thanks Cyrus